provisioner/terraform/executor.go

```diff
@@ -330,18 +331,18 @@ func onlyDataResources(sm tfjson.StateModule) tfjson.StateModule {
 }
 
 // planResources must only be called while the lock is held.
-func (e *executor) planResources(ctx, killCtx context.Context, planfilePath string) (*State, error) {
+func (e *executor) planResources(ctx, killCtx context.Context, planfilePath string) (*State, []byte, error) {
```

Comment:
Just a bit more descriptive. Imo sometimes `[]byte` requires me to follow the thread of code to see what the bytes really are.

```suggestion
func (e *executor) planResources(ctx, killCtx context.Context, planfilePath string) (*State, json.RawMessage, error) {
```